test/message/throw_in_line_with_tabs.js

```diff
@@ -26,9 +26,9 @@ require('../common');
 console.error('before');
 
 (function() {
-	// these lines should contain tab!
-	// eslint-disable-next-line no-throw-literal
-	throw ({ foo: 'bar' });
+  // these lines should contain tab!
```

Comment:
I am a bit confused here. The comment states the tabs are needed here (as the eslint comment above disabling `no-tabs`).